### ISSUE TITLE
cognitive services support?

### ISSUE THREAD

**Issue by kodonnell** [Azure/iot-edge#372]
As per [here](https://azure.microsoft.com/en-us/campaigns/iot-edge/) cognitive services is (to be?) supported. What's the current status of this (or where does it sit on the roadmap)?

**Comment by damonbarry** [comment]
See this blog post: https://azure.microsoft.com/en-us/blog/azure-iot-edge-open-for-developers-to-build-for-the-intelligent-edge/

**Comment by kodonnell** [comment]
So the answer is 'no'? I'm not worried about rolling our own ML - it's whether we can get Cognitive Services on edge devices (which doesn't seem to be addressed by that post).

**Comment by damonbarry** [comment]
Sorry when you said cognitive services I thought you meant the generic concept of cognitive services (which ML falls into). I’m not sure about Azure Cognitive Services; I’ll find out.

**Comment by damonbarry** [comment]
@kodonnell I talked to some folks...  :)

We are working with the Azure Cognitive Services team--in fact we're working with several Azure services. But there has been no public announcement RE the availability of Cognitive Services on IoT Edge, so unfortunately there's nothing to share right now.

I would expect to see a blog post on [Azure's blog](https://azure.microsoft.com/en-us/blog/) as soon as there's something to share.

**Comment by kodonnell** [comment]
Ah, I get a sniff of private announcements! Looks like my original link has been redirected - I would have called that a 'public announcement', though I guess it was jumping the gun.

**Comment by damonbarry** [comment]
Yeah, I can no longer see what you were originally referring to, but I'm guessing it "announced" that we're working to integrate Cognitive Services, but didn't nail down a timeline.

**Comment by kodonnell** [comment]
Maybe [here](https://blogs.microsoft.com/iot/2017/05/10/microsoft-azure-iot-edge-extending-cloud-intelligence-to-edge-devices/)? I.e. 

> Azure IoT Edge enables seamless deployment of cloud services such as ... Artificial Intelligence, including Cognitive Services

**Comment by damonbarry** [comment]
Yep, looks right.
